po/ja.po

```diff
@@ -12078,152 +12238,167 @@ msgid ""
 msgid "`&str`"
-msgstr ""
+msgstr "`&str`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`rust::Str`"
-msgstr ""
+msgstr "`rust::Str`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 #, fuzzy
 msgid "`CxxString`"
-msgstr "文字列（String）"
+msgstr "`CxxString`"
 
 #: src/android/interoperability/cpp/type-mapping.md
-#, fuzzy
 msgid "`std::string`"
-msgstr "文字列（String）"
+msgstr "`std::string`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`&[T]`/`&mut [T]`"
-msgstr ""
+msgstr "`&[T]`/`&mut [T]`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`rust::Slice`"
-msgstr ""
+msgstr "`rust::Slice`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`rust::Box<T>`"
-msgstr ""
+msgstr "`rust::Box<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`UniquePtr<T>`"
-msgstr ""
+msgstr "`UniquePtr<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`std::unique_ptr<T>`"
-msgstr ""
+msgstr "`std::unique_ptr<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`Vec<T>`"
-msgstr ""
+msgstr "`Vec<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`rust::Vec<T>`"
-msgstr ""
+msgstr "`rust::Vec<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`CxxVector<T>`"
-msgstr ""
+msgstr "`CxxVector<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`std::vector<T>`"
-msgstr ""
+msgstr "`std::vector<T>`"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid ""
 "These types can be used in the fields of shared structs and the arguments "
 "and returns of extern functions."
 msgstr ""
+"これらの型は、共有構造体のフィールドと、extern 関数の引数と戻り値で使用できま"
+"す。"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid ""
 "Note that Rust's `String` does not map directly to `std::string`. There are "
 "a few reasons for this:"
 msgstr ""
+"Rust の `String` は `std::string` に直接マッピングされません。これには次のよ"
+"うな理由があります。"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid ""
 "`std::string` does not uphold the UTF-8 invariant that `String` requires."
-msgstr ""
+msgstr "`std::string` は、`String` が必要とする UTF-8 不変条件を維持しません。"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid ""
 "The two types have different layouts in memory and so can't be passed "
 "directly between languages."
 msgstr ""
+"この 2 つの型はメモリ内のレイアウトが異なるため、言語間で直接渡すことはできま"
+"せん。"
 
 #: src/android/interoperability/cpp/type-mapping.md
 msgid ""
 "`std::string` requires move constructors that don't match Rust's move "
 "semantics, so a `std::string` can't be passed by value to Rust."
 msgstr ""
+"`std::string` は、Rust のムーブ セマンティクスと一致しない移動コンストラクタ"
+"を必要とするため、`std::string` を Rust に値で渡すことはできません。"
 
 #: src/android/interoperability/cpp/android-build-cpp.md
 #: src/android/interoperability/cpp/android-cpp-genrules.md
 #: src/android/interoperability/cpp/android-build-rust.md
 #, fuzzy
 msgid "Building in Android"
-msgstr "Android"
+msgstr "Android 向けのビルド: C++"
```

Comment:
Where does this "C++" come from?

Reply:
This is a fuzzy entry, so it needs to be translated

Reply:
I was confused because there is no "C++" in the original message ("Building in Android"). Why was it only added to the Japanese translation result? Was it added for providing better context?

Reply:
In other words, I only expected it to be like below.

```suggestion
msgstr "Android 向けのビルド"
```